### ISSUE TITLE
Camping yield should be halved while on a caravan mission.

### ISSUE THREAD

**Issue by LordMidas** [Battle-Brothers-Legends/Legends-Bugs#124]
Currently, long caravan missions e.g. 3-5 days can yield a large amount of coin and XP all by themselves. This also basically gives the player a free 3-5 day camping spree with basically no opportunity cost. In my opinion this is not balanced. A solution could be to reduce the camping yield while caravaning, to show that you are busy moving with the caravan and don't have all the time in the world to dedicate to your camping tasks. I propose a 0.5x multiplier.

**Comment by Enduriel** [comment]
I think this requires some testing to settle on the final number but agree in principle.

**Comment by TheCutestSkinghoul** [comment]
> I think this requires some testing to settle on the final number but agree in principle.

Tested this just now, we can create a flag like 'on caravan contract' and then change gather rates from there.

I reduced them all by ~20% and it feels better but I'm worried about lack of repairing and healing between fighters on harder difficulties.

Maybe it's for the best. It's quite a large nerf for the player and I think it'll get some pushback
